python/gink/crud.py

```diff
@@ -0,0 +1,149 @@
+"""
+The WSGI API application for gink operations.
+
+To run this application, pass --wsgi gink.crud.app when starting gink.
+
+Set auth key with env AUTH_TOKEN.
+
+"""
+
+from json import loads, dumps
+from os import environ
+
+from .impl.directory import Directory
+
+
+class Crud():
```

Comment:
I'd prefer crud to just be a function rather than a class that gets instantiated.  It's okay to do what you're doing in the `__init__` method every time the WSGI function gets called.  Most (all?) of the other methods don't actually use `self` and could just be regular functions.  And to serve as an example of a WSGI application I think it's best for it to be as conceptually simple as possible.